zulipterminal/ui_tools/color.py

```diff
@@ -0,0 +1,13 @@
+from re import match, ASCII
+
+
+def canonicalize_color(color: str) -> str:
```

Comment:
@alex-ozdemir I would also suggest adding test cases for it. You can look at the existing tests for getting an overview of how it should look like.